lib/components/SInputYMD.vue

```diff
@@ -302,6 +302,7 @@ function createRequiredTouched(): boolean[] {
 
 .input {
   line-height: 24px;
+  font-family: var(--input-value-font-family, var(--font-family-number));
```

Comment:
Added `--font-family-number` as a fallback. [`SInputHMS`](https://github.com/globalbrain/sefirot/blob/main/lib/components/SInputHMS.vue#L311) has it, is there any specific reason this didn't?

Reply:
Nope. I think because `SInputHMS` is older and I've forgot to add support to it 😅